Sources/NIOTransportServices/StateManagedChannel.swift

```diff
@@ -45,22 +45,22 @@ internal enum ChannelState<ActiveSubstate: ActiveChannelSubstate> {
     /// Unlike every other one of these methods, this one has a side-effect. This is because
     /// it's impossible to correctly be in the reigstered state without verifying that
     /// registration has occurred.
-    fileprivate mutating func register(eventLoop: NIOTSEventLoop, channel: Channel) throws {
+    mutating func register(eventLoop: NIOTSEventLoop, channel: Channel) throws {
         guard case .idle = self else {
             throw NIOTSErrors.InvalidChannelStateTransition()
         }
         try eventLoop.register(channel)
         self = .registered
     }
 
-    fileprivate mutating func beginActivating() throws {
+    mutating func beginActivating() throws {
         guard case .registered = self else {
             throw NIOTSErrors.InvalidChannelStateTransition()
         }
         self = .activating
     }
 
-    fileprivate mutating func becomeActive() throws {
+    mutating func becomeActive() throws {
```

Comment:
Is it actually necessary to remove the `fileprivate`s here?

Reply:
You're right, it's not. I guess it lingered around from some other code I wrote.

Reply:
Thanks, it is now